core/tauri/src/ipc/protocol.rs

```diff
@@ -42,11 +47,24 @@ pub fn get<R: Runtime>(manager: WindowManager<R>, label: String) -> UriSchemePro
     match *request.method() {
       Method::POST => {
         if let Some(window) = manager.get_window(&label) {
-          match parse_invoke_request(&manager, request) {
+          match parse_invoke_request(invoke_id, &manager, request) {
             Ok(request) => {
+              let request_span =
+                tracing::trace_span!("ipc::request::handle", id = request.id.0, cmd = request.cmd);
+
+              let id = request.id;
+
               window.on_message(
                 request,
                 Box::new(move |_window, _cmd, response, _callback, _error| {
+                  let _span = tracing::trace_span!(
+                    parent: &request_span,
+                    "ipc::request::respond",
```

Comment:
This span ID is way to similar to `ipc::request::response`, It is also duplicating the data sent by the next span